### ISSUE TITLE
Docs: metrics update 5.0 to 5.1

### ISSUE THREAD

**Issue by tzach** [scylladb/scylladb#11377]

**Comment by tzach** [comment]
@amnonh ping

**Comment by amnonh** [comment]
|New in 5.1|Description|
|---------------|---------------|
|scylla_cache_rows_compacted_with_tombstones|Number of rows dropped in cache by a tombstone write|
scylla_cache_rows_dropped_by_tombstones|total number of rows in memtables which were dropped during cache update on memtable flush
scylla_commitlog_active_allocations|Current number of active allocations.
scylla_commitlog_blocked_on_new_segment|Number of allocations blocked on acquiring new segment.
scylla_commitlog_bytes_flush_requested|Counts number of bytes requested to be flushed (persisted).
scylla_commitlog_bytes_released|Counts number of bytes released from disk. (Deleted/recycled)
scylla_compaction_manager_completed_compactions|Holds the number of completed compaction tasks.
scylla_compaction_manager_failed_compactions|Holds the number of failed compaction tasks.
scylla_compaction_manager_normalized_backlog|Holds the sum of normalized compaction backlog for all tables in the system. Backlog is normalized by dividing backlog by shard's available memory.
scylla_compaction_manager_postponed_compactions|Holds the number of tables with postponed compaction.
scylla_compaction_manager_validation_errors|Holds the number of encountered validation errors.
scylla_cql_select_parallelized|Counts the number of parallelized aggregation SELECT query executions.
scylla_database_total_reads_rate_limited|Counts read operations which were rejected on the replica side because the per-partition limit was reached.
scylla_database_total_writes_rate_limited|Counts write operations which were rejected on the replica side because the per-partition limit was reached.
scylla_forward_service_requests_dispatched_to_other_nodes|how many forward requests were dispatched to other nodes
scylla_forward_service_requests_dispatched_to_own_shards|how many forward requests were dispatched to local shards
scylla_forward_service_requests_executed|how many forward requests were executed
scylla_gossip_live|How many live nodes the current node sees
scylla_gossip_unreachable|How many unreachable nodes the current node sees
scylla_io_queue_adjusted_consumption|Consumed disk capacity units adjusted for class shares and idling preemption
scylla_io_queue_consumption|Accumulated disk capacity units consumed by this class; an increment per-second rate indicates full utilization
scylla_io_queue_total_split_bytes|Total number of bytes split
scylla_io_queue_total_split_ops|Total number of requests split
scylla_per_partition_rate_limiter_allocations|Number of times a entry was allocated over an empty/expired entry.
scylla_per_partition_rate_limiter_failed_allocations|Number of times the rate limiter gave up trying to allocate.
scylla_per_partition_rate_limiter_load_factor|Current load factor of the hash table (upper bound, may be overestimated).
scylla_per_partition_rate_limiter_probe_count|Number of probes made during lookups.
scylla_per_partition_rate_limiter_successful_lookups|Number of times a lookup returned an already allocated entry.
scylla_reactor_aio_outsizes|Total number of aio operations that exceed IO limit
scylla_schema_commitlog_active_allocations|Current number of active allocations.
scylla_schema_commitlog_alloc|Holds the number of not closed segments that still have some free space. This value should not get too high.
scylla_schema_commitlog_allocating_segments|Counts number of times a new mutation has been added to a segment. Divide bytes_written by this value to get the average number of bytes per mutation written to the disk.
scylla_schema_commitlog_blocked_on_new_segment|Number of allocations blocked on acquiring new segment.
scylla_schema_commitlog_bytes_flush_requested|Counts number of bytes requested to be flushed (persisted).
scylla_schema_commitlog_bytes_released|Counts number of bytes released from disk. (Deleted/recycled)
scylla_schema_commitlog_bytes_written|Counts number of bytes written to the disk. Divide this value by "alloc" to get the average number of bytes per mutation written to the disk.
scylla_schema_commitlog_cycle|Counts number of commitlog write cycles - when the data is written from the internal memory buffer to the disk.
scylla_schema_commitlog_disk_active_bytes|Holds size of disk space in bytes used for data so far. A too high value indicates that we have some bottleneck in the writing to sstables path.
scylla_schema_commitlog_disk_slack_end_bytes|Holds size of disk space in bytes unused because of segment switching (end slack). A too high value indicates that we do not write enough data to each segment.
scylla_schema_commitlog_disk_total_bytes|Holds size of disk space in bytes reserved for data so far. A too high value indicates that we have some bottleneck in the writing to sstables path.
scylla_schema_commitlog_flush|Counts number of times the flush() method was called for a file.
scylla_schema_commitlog_flush_limit_exceeded|Counts number of times a flush limit was exceeded. A non-zero value indicates that there are too many pending flush operations (see pending_flushes) and some of them will be blocked till the total amount of pending flush operations drops below 5.
scylla_schema_commitlog_memory_buffer_bytes|Holds the total number of bytes in internal memory buffers.
scylla_schema_commitlog_pending_allocations|Holds number of currently pending allocations. A non-zero value indicates that we have a bottleneck in the disk write flow.
scylla_schema_commitlog_pending_flushes|Holds number of currently pending flushes. See the related flush_limit_exceeded metric.
scylla_schema_commitlog_requests_blocked_memory|Counts number of requests blocked due to memory pressure. A non-zero value indicates that the commitlog memory quota is not enough to serve the required amount of requests.
scylla_schema_commitlog_segments|Holds the current number of segments.
scylla_schema_commitlog_slack|Counts number of unused bytes written to the disk due to disk segment alignment.
scylla_schema_commitlog_unused_segments|Holds the current number of unused segments. A non-zero value indicates that the disk write path became temporary slow.
scylla_sstables_pi_auto_scale_events|Number of promoted index auto-scaling events
scylla_storage_proxy_coordinator_read_rate_limited|number of read requests which were rejected by replicas because rate limit for the partition was reached.
scylla_storage_proxy_coordinator_write_rate_limited|number of write requests which were rejected by replicas because rate limit for the partition was reached.

**Comment by tzach** [comment]
Thanks @amnonh 
what about update / deletes?

**Comment by amnonh** [comment]
@tzach there are none
